docusaurus/docs/ruby/rspec.mdx

```diff
@@ -105,6 +108,37 @@ export KNAPSACK_PRO_TEST_FILE_EXCLUDE_PATTERN={spec/requests/**/*_spec.rb,spec/a
 bundle exec rake "knapsack_pro:queue:rspec"
 ```
 
+## Load code only once for a specific type of specs using `when_first_matching_example_defined`
```

Comment:
```suggestion
## Load code only once for a specific type of specs
```

Reply:
Did anybody request this? I'd like to see how they phrased it. I'd like to change the copy here a bit.

Reply:
Yes. Please see: https://knapsackpro.slack.com/archives/C05RJKCQ42D/p1706891273447509?thread_ts=1702626494.818529&cid=C05RJKCQ42D